app/src/androidTest/assets/features/visualVerification.feature

```diff
@@ -0,0 +1,18 @@
+Feature: VisualVerification
+  Do several things on the VisualVerificationActivity
+
+  Scenario Outline: Fill out a correct pattern
+    Given I am viewing the visual verification activity
+    When I press button <first_button>
+    And I press button <second_button>
+    And I press button <third_button>
+    And I press button <fourth_button>
+    And I press button <fifth_button>
+    Then I leave the visual verification activity
```

Comment:
Should you always leave this activity, even in the case your sequence wasn't the right one? Or is this for the person that initiates the connection?